### ISSUE TITLE
Tests currently failing

### ISSUE THREAD

**Issue by dkoudrin** [selenide-examples/selenoid-sample#2]
Hi,
I pulled down the project and ran it with mvn test, 2 out of 3 tests failed - is that expected or have they become outdated?

e.g.
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   ClipboardTest.prepare:24 Element not found {.snigel-cmp-framework}
Expected: visible
Screenshot: file:/C:/Users/DmitriKoudrin/Documents/selenoid-sample/target/surefire-reports/1667767750994.0.png
Page source: file:/C:/Users/DmitriKoudrin/Documents/selenoid-sample/target/surefire-reports/1667767750994.0.html
Timeout: 4 s.
Caused by: NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":".snigel-cmp-framework"}
[ERROR]   ClipboardTest.prepare:24 Element not found {.snigel-cmp-framework}
Expected: visible
Screenshot: file:/C:/Users/DmitriKoudrin/Documents/selenoid-sample/target/surefire-reports/1667767755731.1.png
Page source: file:/C:/Users/DmitriKoudrin/Documents/selenoid-sample/target/surefire-reports/1667767755731.1.html
Timeout: 4 s.
Caused by: NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":".snigel-cmp-framework"}
[INFO]
[ERROR] Tests run: 3, Failures: 2, Errors: 0, Skipped: 0

**Comment by asolntsev** [comment]
@dkoudrin Of course, this test may become outdated at any moment because it uses 3rd party site www.w3schools.com

Its developers can change its behaviour at any moment.

Ideally, we would need to rewrite our tests to be independent from 3rd party resources.
